src/abstractdataframe/abstractdataframe.jl

```diff
@@ -551,11 +551,13 @@ Indexes of complete cases (rows without NA's)
 
 ```julia
 nonunique(df::AbstractDataFrame)
+nonunique(df::AbstractDataFrame, columns)
 ```
 
 ### Arguments
 
 * `df` : the AbstractDataFrame
+* `columns` : specifies the columns to compare
```

Comment:
"specifies" isn't useful here. OTOH, it would be useful to precise that columns can be either a symbol, an integer, or vectors of these types.